package.json

```diff
@@ -27,31 +27,31 @@
     "npm": ">=2.1.5"
   },
   "devDependencies": {
-    "asset-builder": "^1.0.2",
-    "browser-sync": "^2.7.1",
-    "del": "^1.1.1",
-    "gulp": "^3.8.11",
-    "gulp-autoprefixer": "^2.1.0",
+    "asset-builder": "^1.1.0",
+    "browser-sync": "^2.7.6",
+    "del": "^1.2.0",
+    "gulp": "^3.9.0",
+    "gulp-autoprefixer": "^2.3.1",
     "gulp-changed": "^1.2.1",
     "gulp-concat": "^2.5.2",
     "gulp-flatten": "0.0.4",
     "gulp-if": "^1.2.5",
     "gulp-imagemin": "^2.2.1",
-    "gulp-jshint": "^1.9.4",
-    "gulp-less": "^3.0.2",
-    "gulp-minify-css": "^1.0.0",
-    "gulp-plumber": "^1.0.0",
-    "gulp-rename": "^1.2.0",
-    "gulp-rev": "^3.0.1",
-    "gulp-sass": "^2.0.0",
-    "gulp-sourcemaps": "^1.5.1",
-    "gulp-uglify": "^1.1.0",
-    "imagemin-pngcrush": "^4.0.0",
-    "jshint-stylish": "^1.0.1",
-    "lazypipe": "^0.2.2",
+    "gulp-jshint": "^1.11.0",
+    "gulp-less": "^3.0.3",
+    "gulp-minify-css": "^1.1.5",
+    "gulp-plumber": "^1.0.1",
+    "gulp-rename": "^1.2.2",
+    "gulp-rev": "^4.0.0",
```

Comment:
4.0.0 doesn't seem to break much https://github.com/sindresorhus/gulp-rev/compare/2f9d798c4d8f4c91bec215df68bfcc3850caa34f...4.0.0